File: backend/DataAccess/DaoGenerated/システム共有/tt_afmemoDto.cs
// *******************************************************************
// 業務名称　: 健康管理システム
// 機能概要　: DTO定義
//             メモテーブル
// 作成日　　: 2023.01.23
// 作成者　　: 屠
// 変更履歴　:
// *******************************************************************
namespace BCC.Affect.DataAccess
{
    public class tt_afmemoDto : DaEntityModelBase
    {
		public const string TABLE_NAME = "tt_afmemo";
		public string atenano { get; set; }                       //宛名番号
		public int memoseq { get; set; }                          //メモシーケンス
		public string jigyokbn { get; set; }                      //メモ事業コード
		public string juyodo { get; set; }                        //重要度
		public string? title { get; set; }                        //件名（タイトル）
		public string memo { get; set; }                          //メモ（フリーテキスト）
		public string? kigenymd { get; set; }                     //期限日
		public string? regsisyo { get; set; }                     //登録支所
		public string reguserid { get; set; }                     //登録ユーザーID
		public DateTime regdttm { get; set; }                     //登録日時
		public string upduserid { get; set; }                     //更新ユーザーID
		public DateTime upddttm { get; set; }                     //更新日時
    }
}

File: backend/JbdGjsService/Db/DaoGenerated/システム共有/tt_afmemoDto.vb
' *******************************************************************
' 業務名称　: 互助防疫システム
' 機能概要　: DTO定義
'             メモテーブル
' 作成日　　: 2024.07.23
' 作成者　　: 
' 変更履歴　:
' *******************************************************************
Namespace JBD.GJS.Db
    Public Class tt_afmemoDto
        Inherits DaEntityModelBase
        Public Const TABLE_NAME As String = "tt_afmemo"
        Public Property atenano As String                       '宛名番号
        Public Property memoseq As Integer                          'メモシーケンス
        Public Property jigyokbn As String                      'メモ事業コード
        Public Property juyodo As String                        '重要度
        Public Property title As String                        '件名（タイトル）
        Public Property memo As String                          'メモ（フリーテキスト）
        Public Property kigenymd As String                     '期限日
        Public Property regsisyo As String                     '登録支所
        Public Property reguserid As String                     '登録ユーザーID
        Public Property regdttm As Date                     '登録日時
        Public Property upduserid As String                     '更新ユーザーID
        Public Property upddttm As Date                     '更新日時
    End Class
End Namespace
